### PR TITLE
Adding d3 viz libraries

### PR DESCRIPTION
**Pull request by cjekal**
adding d3 viz library and drawing the attorneys/canonical groups on Tableau.

### CODE REVIEW

File: GetData/getdata.html [hunk 1]
```diff
@@ -1,59 +1,98 @@
...
-            </thead>
-            <tbody>
-            </tbody>
-          </table>
-        </div>
+  <!-- lodash -->
+  <script src="https://cdn.jsdelivr.net/npm/lodash@4.17.11/lodash.min.js"></script>
+
+  <!-- D3 -->
+  <script src="https://d3js.org/d3.v6.min.js"></script>
```
Comment (cjekal) [inline GetData/getdata.html:21]: @echasin I know you requested that I download these and serve these from the web server and not from CDNs, so I'll work on that next.

File: GetData/getdata.html [hunk 2]
```diff
@@ -1,59 +1,98 @@
...
+      pointer-events: none;
+      text-anchor: middle;
+      font: 10px sans-serif;
+    }
+
+    .d3-visualization {
+      width: 960;
+      height: 600;
+    }
+  </style>
```
Comment (cjekal) [inline GetData/getdata.html:50]: being lazy about stylesheets

File: GetData/getdata.js [hunk 3]
```diff
@@ -144,4 +160,112 @@
...
+        "group": 1
+      }
+    });
+    var links = _.map(data, row => {
+      return {
+        "source": "canonical" + row.canonical_component_id,
+        "target": "source" + row.attorney_id_target,
+        "value": 1
+      }
+    });
```
Comment (cjekal) [inline GetData/getdata.js:196]: using `lodash` to generate our canonical nodes, attorney nodes and links.

File: GetData/getdata.html (hunk 1 above)
Comment (echasin) [reply GetData/getdata.html:21]: Approved... Much better if we dont use CDN for this project

**Pull request merged**